src/applications/my-education-benefits/config/form.js

```diff
@@ -780,12 +788,98 @@ const formConfig = {
                       }
                     },
                   ],
+                  'ui:options': {
+                    replaceSchema: formData => {
+                      const { livesOnMilitaryBase } = formData[
+                        'view:mailingAddress'
+                      ];
+                      const { country } = formData[
+                        'view:mailingAddress'
+                      ].address;
+
+                      if (
+                        // formData.showMEBMailingAddressForeign &&
```

Comment:
Was this feature flag commented out for testing?

Reply:
Still working on this, this can be ignored